app/assets/javascripts/govuk_publishing_components/components/table-filter.js

```diff
@@ -0,0 +1,44 @@
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (Modules) {
+  function TableFilter ($module) {
+    this.$module = $module
+    this.searchInput = $module.querySelector('input[name="filter"]')
+    this.tableRows = $module.querySelectorAll('.js-govuk-table__row')
+    this.filter = $module.querySelector('.js-gem-c-table__filter')
+    this.message = $module.querySelector('.js-gem-c-table__message')
+    this.hiddenRowClass = 'govuk-table__row--hidden'
+    this.hiddenMessageClass = 'gem-c-table__message--hidden'
+  }
+
+  TableFilter.prototype.init = function () {
+    this.$module.updateRows = this.updateRows.bind(this)
+    this.filter.style.display = 'block'
+    this.searchInput.addEventListener('input', this.$module.updateRows)
+  }
+
+  // Reads value of input and filters content
+  TableFilter.prototype.updateRows = function () {
+    var value = this.searchInput.value
+    var hiddenRowClass = this.hiddenRowClass
+    var hiddenRows = 0
+
+    this.tableRows.forEach(function (row) {
+      row.classList.remove(hiddenRowClass)
```

Comment:
Unfortunately, IE11 doesn't support `forEach` on a NodeList https://developer.mozilla.org/en-US/docs/Web/API/NodeList/forEach and it doesn't look like we have a polyfill here included... so I think you'll have to use a for loop here (or convert the NodeList to an Array).

<img width="1579" alt="ie11-screen-capture" src="https://user-images.githubusercontent.com/87758239/173339740-a0391032-507d-4313-a185-6e8e32c208e1.png">

Reply:
Oops, my bad - I kinda thought about that then forgot. Also didn’t test cross browser! I’ve rewritten that now and tested it more fully.